types/ckeditor__ckeditor5-utils/src/dom/emittermixin.d.ts

```diff
@@ -1,32 +1,66 @@
-import { DomEventData } from "@ckeditor/ckeditor5-engine";
 import { Emitter as BaseEmitter, EmitterMixinDelegateChain } from "../emittermixin";
 import EventInfo from "../eventinfo";
 import { PriorityString } from "../priorities";
 
 export interface Emitter extends BaseEmitter {
-    delegate(...events: string[]): EmitterMixinDelegateChain;
-    fire(eventOrInfo: string | EventInfo, ...args: any[]): any;
-    listenTo(
-        emitter: Emitter,
-        event: string,
-        callback: (info: EventInfo, data: DomEventData) => void,
```

Comment:
This callback signature was incorrectly copied to all `listenTo` and `on` methods, but here it was actually correct.

Reply:
I've reverted this.